### PR TITLE
[GAL-4122] Mobile sign up with email

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

1. User able to create a gallery account with email
2. Added new email onboarding design
3. User able to add evm wallet to existing account

This PR need to be merged first: https://github.com/gallery-so/gallery/tree/jakz/sign-up-wallet

### Demo or Before/After Pics

**Sign up with email**

https://github.com/gallery-so/gallery/assets/4480258/5fbaed7a-f952-4fb4-8337-dbe541daaab9

**Sign in with email**

https://github.com/gallery-so/gallery/assets/4480258/fcec1504-a9e7-4e4a-99ac-241b6123a635

### Edge Cases

1. Create a new account without selecting PFP
2. Create a new account with PFP + bio
3. Check invalid email address
4. Dark mode
5. Sign in back with new created account with email

### Testing Steps

1. Enter the new onboarding screen
2. Select sign in with email
3. If your email is not registered, it will redirect to onboarding screen (username)
4. You're able to connect your wallet when selecting the PFP
5. Redirect to feed

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/screens/Login/OnboardingEmailScreen.tsx
Comment (jakzaizzat) [inline apps/mobile/src/screens/Login/OnboardingEmailScreen.tsx:1]: Created a new screen for email to separated with old design. 

We could totally toss the old design and replaced with this screen when we launch

File: apps/mobile/src/screens/Onboarding/OnboardingProfileBioScreen.tsx [hunk 1]
```diff
@@ -77,6 +97,72 @@ export function OnboardingProfileBioScreen() {
...
+
+      if (!isSyncing) {
+        syncTokens('Ethereum');
+      }
+
+      bottomSheet.current?.dismiss();
+
+      navigation.navigate('OnboardingNftSelector', {
+        page: 'ProfilePicture',
+      });
```
Comment (jakzaizzat) [inline apps/mobile/src/screens/Onboarding/OnboardingProfileBioScreen.tsx:151]: Is the increment token sync included when the existing user just added a new wallet? 

If yes, I can totally toss it later

File: apps/mobile/src/screens/Login/OnboardingEmailScreen.tsx
Comment (kaitoo1) [reply apps/mobile/src/screens/Login/OnboardingEmailScreen.tsx:1]: good plan 🧠

File: apps/mobile/src/screens/Onboarding/OnboardingProfileBioScreen.tsx (hunk 1 above)
Comment (kaitoo1) [reply apps/mobile/src/screens/Onboarding/OnboardingProfileBioScreen.tsx:151]: hm im not sure! lets discuss on slack

File: apps/mobile/src/components/Login/WalletSelectorBottomSheet.tsx [hunk 2]
```diff
@@ -19,88 +20,114 @@ import { Typography } from '../Typography';
...
 ) {
   const { bottom } = useSafeAreaPadding();
 
+  const { open, isConnected, provider } = useWalletConnectModal();
   const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
 
   const { animatedHandleHeight, animatedSnapPoints, animatedContentHeight, handleContentLayout } =
     useBottomSheetDynamicSnapPoints(SNAP_POINTS);
 
+  const handleConenctWallet = useCallback(async () => {
```
Comment (kaitoo1) [inline apps/mobile/src/components/Login/WalletSelectorBottomSheet.tsx:56]: small typo `Conenct`

**Review by kaitoo1 — APPROVED** [review]
nice one jakz!!!!

**Pull request merged**